app/web/index.html

```diff
@@ -23,37 +23,16 @@
   <!-- iOS meta tags & icons -->
   <meta name="apple-mobile-web-app-capable" content="yes">
   <meta name="apple-mobile-web-app-status-bar-style" content="black">
-  <meta name="apple-mobile-web-app-title" content="manvsim">
+  <meta name="apple-mobile-web-app-title" content="MANVSim">
   <link rel="apple-touch-icon" href="icons/Icon-192.png">
 
   <!-- Favicon -->
   <link rel="icon" type="image/png" href="favicon.png"/>
 
-  <title>manvsim</title>
+  <title>MANVSim</title>
   <link rel="manifest" href="manifest.json">
-
-  <script>
-    // The value below is injected by flutter build, do not touch.
-    const serviceWorkerVersion = null;
-  </script>
-  <!-- This script adds the flutter initialization JS code -->
-  <script src="flutter.js" defer></script>
 </head>
 <body>
-  <script>
-    window.addEventListener('load', function(ev) {
-      // Download main.dart.js
-      _flutter.loader.loadEntrypoint({
-        serviceWorker: {
-          serviceWorkerVersion: serviceWorkerVersion,
-        },
-        onEntrypointLoaded: function(engineInitializer) {
```

Comment:
I'm not understand this changes in detail, but it seems that they cause the Flutter web app to fail at startup with this error message (from chrome console, other browser similar):

`Refused to execute script from 'http://localhost:61028/flutter_bootstrap.js' because its MIME type ('text/html') is not executable, and strict MIME type checking is enabled.`

Reply:
I don't understand these changes in detail either.  
But i always got a deprecated warning for those methods when starting the web and it was recommended to regenerate these files.  
Which Flutter version do you use? I use 3.24.1 and it runs just fine. It also runs in the docker container and GitHub pipeline. @lukas0820

Reply:
It seems we were using different Flutter versions to build the app. I've now upgraded to version 3.24.1, which resolved the issue